commands-yml/commands/context/get-context.yml

```diff
@@ -0,0 +1,81 @@
+---
+name: Get Current Context
+short_description: Get the current context in which Appium is running
+description:
+  >
+    Retrieve the current context. This can be either `NATIVE_APP` for the native
+    context, or a web view context, which will be:
+
+      * iOS - `WEBVIEW_<id>`
+      * Android - `WEBVIEW_<package name>`
+
+
+    For information on contexts, see Appium's [hybrid automation docs](/docs/en/writing-running-appium/web/hybrid.md).
+
+example_usage:
+  java:
+    |
+      String context = driver.getContext();
+  python:
+    |
+      context = driver.current_context
+      # of
+      context = driver.context
+  javascript_wd:
+    |
+      let context = await driver.currentContext();
+  javascript_wdio:
+    |
+      let context = driver.context();
+  ruby:
+    |
+      context = @driver.current_context
+  php:
+    |
+      $context = $driver->context();
+  csharp:
+    |
+      // TODO C# sample
+
+client_docs:
+  java: "https://github.com/appium/java-client"
```

Comment:
http://appium.github.io/java-client/io/appium/java_client/AppiumDriver.html#getContext--

Reply:
Hmm. That's definitely some pointless documentation.